ext/bin/load_test/load_test.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+from locust import HttpUser, task, between
+import yaml
+import json
+import os
+
+class QuickstartUser(HttpUser):
+    wait_time = between(1, 2.5)
```

Comment:
In order to make the runs predictable I would go with a fixed value for this (e.g. `1`) or maybe we can eliminate it altogether to submit commands as fast as possible.